webapp/lib/charts.dart

```diff
@@ -130,25 +133,29 @@ class DailyTimeseriesLineChartView {
             ..distribution = 'linear'
             ..bounds = 'ticks'
             ..time = (new chartjs.TimeScale(unit: 'day'))
-        ],
-        yAxes: [
-          new chartjs.ChartYAxe()
-            ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
+      ],
+      yAxes: [
+        new chartjs.ChartYAxe()
+          ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
```

Comment:
accidental whitespace change? please revert

Reply:
Removed the whitespace